l10n.yaml

```
@@ -0,0 +1,5 @@
+arb-dir: lib/l10n
+template-arb-file: app_en.arb
+output-localization-file: app_localizations.dart
+output-dir: lib/l10n/generated
+synthetic-package: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)